cliquet/__init__.py

```diff
@@ -71,6 +71,7 @@
     'cliquet.sentry_url': None,
     'cliquet.sentry_projects': '',
     'cliquet.statsd_url': None,
+    'cliquet.statsd_prefix': 'cliquet'
```

Comment:
I'm not sure for the env, since I believe we have different stacks depending the env used.